aptos-move/framework/src/built_package.rs

```diff
@@ -68,6 +68,10 @@ pub struct BuildOptions {
     pub skip_fetch_latest_git_deps: bool,
     #[clap(long)]
     pub bytecode_version: Option<u32>,
+    #[clap(long)]
+    pub skip_attribute_checks: bool,
+    #[clap(skip)]
+    pub known_attributes: BTreeSet<String>,
```

Comment:
I don't think this need to be configurable, since the BuildConfig is only for Aptos, and in Aptos we wont use command line options to define allowed attributes. Looking at existing languages with attributes (Java/C#/Python come to mind), none of them have attributes defined this way. I'm worried we are creating technical debt here by adding this option which we now have to maintain forever. Can we remove it and hardwire the set of known attributes? I also wouldn't mind if, for everything else then Aptos (e.g. evm or async flavor), we just use --skip-attribute-checks.

Reply:
The desire is not to have it be configurable by the user, but to make it configurable per Move dialect (base, Aptos, EVM, Async).  As we discussed, this may be overkill for the current use case, but can serve as a guide for some more interesting dialect changes later.